common/src/main/java/org/opensearch/ml/common/agent/MLToolSpec.java

```diff
@@ -26,25 +26,29 @@ public class MLToolSpec implements ToXContentObject {
     public static final String TOOL_NAME_FIELD = "name";
     public static final String DESCRIPTION_FIELD = "description";
     public static final String PARAMETERS_FIELD = "parameters";
+    public static final String INCLUDE_OUTPUT_IN_AGENT_RESPONSE = "include_output_in_agent_response";
 
     private String type;
     private String name;
     private String description;
     private Map<String, String> parameters;
+    private Boolean includeOutputInAgentResponse;
```

Comment:
Is it possible the `includeOutputInAgentResponse` is null? If no, how about use `boolean` and set default value for `includeOutputInAgentResponse`? Otherwise line 61 may fail if `includeOutputInAgentResponse` is null.

Reply:
Given it is `Boolean` line 61 should not result in NPE, however it is a good suggestion to use `boolean`. Updated code.